### ISSUE TITLE
Add revalidation to relay post action such that channel items on the home page are refetched

### ISSUE THREAD

**Issue by salieflewis** [1ifeworld/river#219]
Whenever a new item is added, the channel cards on the index route should reflect how many items are in that channel. Currently that fetch request is being cached so we see a stale value.

**Comment by salieflewis** [comment]
One option:

```
import sdk from '../client'
import { unstable_noStore as noStore } from 'next/cache'

export async function getAllChannels() {
  // `noStore()` here prevents the response from being cached
  noStore()
  const response = await sdk.allChannels()

  return {
    channels: response.channels,
  }
}
```

**Comment by salieflewis** [comment]
Opening this up again because want to grab a cached version of the index route here instead of a fresh fetch request every time you hit the home page. This speeds up performance a lot, and we should find a more creative way of reaching for/validating this data.

**Comment by salieflewis** [comment]
Fix introduced in #279 doesn't seem to work in prod.

**Comment by salieflewis** [comment]
Related issue: https://github.com/vercel/next.js/issues/59969
